src/screens/Profile/Profile.jsx

```diff
@@ -124,23 +161,54 @@ const Profile = () => {
           </div>
         </div>
       </div>
-      <Modal
-        title={t("screens.profile.modals.changePassword.heading")}
-        open={showPasswordModal}
+
+      <ConfirmDialog
```

Comment:
add a submit label property.
![image](https://user-images.githubusercontent.com/60048565/118407300-80ecbe00-b680-11eb-887c-2cd93a240e68.png)

Reply:
You mean showSnack?

Reply:
The button says "Delete" instead of "Submit".

Reply:
Oh, I didn't even recognize this